servicecontrol/servicecontrol-instances/hardware.md

```diff
@@ -0,0 +1,61 @@
+---
+title: ServiceControl Hardware Considerations
+summary: Describes performance benchmarks performed for ServiceControl
+reviewed: 2018-10-15
+---
+
+ServiceControl as an application can be used to process the entire message load of a system. This articles provides general guidelines, recommendations, and performance benchmarks to help determine the resources to provide for a production environment. To identify the hardware specifications for any system environment a combination of testing with the system and the information provided below will need to be used.
+
+## General Recommendations
+
+* Use dedicated hardware for ServiceControl in production.
+* 6GB of RAM minimum
+* 2Ghz quad core CPU or better
+* [Database Path](/servicecontrol/creating-config-file.md#host-settings-servicecontroldbpath) is located on disks suitable for low latency write operations (fiber, solid state drives, raid 10), with a recommended IOPS of at least 7500.
+
+### Server Performance Monitoring
+
+Due to changes in the system it supports the requirements for a server hosting ServiceControl can change over time. It is highly recommended that monitoring of the CPU, RAM, disk IO, and network IO for the server running ServiceControl be included.
+
+## Benchmark Data
+
+Version 3.0.0 of ServiceControl was tested to validate performance improvements made between version 2 and version 3. 
+
+NOTE: The test harness used is a highly simplified test. It is highly recommended to run performance tests with realistic message loads to validate baseline hardware requirements. This benchmark data is only meant as a point of reference to assist with determining dedicated ServiceControl server requirements.
+
+### Hardware Used
+
+* 16 vCPU
```

Comment:
Should we give examples of VM sizes or better leave it out because it might be outdated?

Reply:
We considered it but decided to leave it out. We didn't want to tie it so closely to Azure.